userapp/src/components/user/UserForm.js

```diff
@@ -35,49 +60,43 @@ const UserForm = ({ initialUser, onSubmit}) => {
     const handleInputChange = (event) => {
         const { name, value } = event.target
         setUser({ ...user, [name]: value })
-        // validate(name, value)
+        validate(name, value)
     }
 
     /**
-     * TODO: Function will need to change
      * @param {string} name represent accessor of the object
      * @param {string} value represent the keyboard input value from the event object
      */
-    // const validate = (name, value) => {
-    //     switch(name) {
-    //         case "FirstName":
-    //             if(value.length > 3) {
-    //                 console.log("no name")
-    //                 setErrors({ ...user, [name]: ""})
-    //             } else {
-    //                 setErrors({ ...user, [name]: "First name cannot be empty"})
-    //             }
-    //             break
-    //         case "LastName":
-    //             if(value.length > 3) {
-    //                 setErrors({ ...user, [name]: ""})
-    //             } else {
-    //                 setErrors({ ...user, [name]: "Last name cannot be empty"})
-    //             }
-    //             break
-    //         case "DateOfBirth":
-    //             if(value.length === 8) {
-    //                 setErrors({ ...user, [name]: ""})
-    //             } else {
-    //                 setErrors({ ...user, [name]: "Date of Birth should be  MMDDYYYY"})
-    //             }
-    //             break
-    //         case "Phone":
-    //             if(value.length === 10) {
-    //                 setErrors({ ...user, [name]: ""})
-    //             } else {
-    //                 setErrors({ ...user, [name]: "Phone must have length of 10"})
-    //             }
-    //             break
-    //         default:
-    //             break
-    //     }
-    // }
+    const validate = (name, value) => {
```

Comment:
we can simplify these cases to all call one method of validation (code simplification)